fraud-proof/proof/state/stack.go

```diff
@@ -90,7 +90,7 @@ func (st *Stack) Back(n int) *uint256.Int {
 }
 
 func (st *Stack) HashAfterPops(n int) common.Hash {
-	return st.hash[st.Len()-n]
+	return st.hash[len(st.hash)-n]
```

Comment:
Is it possible that `len(st.hash)` is less than `n`?

Reply:
popNum is determined by opcode stack input, i.e, op: ADD
<img width="1064" alt="image" src="https://github.com/mantlenetworkio/mantle/assets/24247813/8c7839f6-c019-411e-bb3d-74422d3af7c4">
it will pop 2 element, and full stack will always be fit pop depth refer to opcode or else evm will revert execution